Sources/Danger/BitBucketServerDSL.swift

```diff
@@ -298,23 +286,42 @@ public struct BitBucketServerPR: Decodable, Equatable {
     public let isLocked: Bool
 
     /// The creator of the PR
-    public let author: BitBucketServerAuthor
+    public let author: Participant
 
     /// People requested as reviewers
-    public let reviewers: [BitBucketServerUser]
+    public let reviewers: [Reviewer]
 
     /// People who have participated in the PR
-    public let participants: [BitBucketServerAuthor]
+    public let participants: [Participant]
 
     // MARK: - BitBucketServerAuthor
 
-    public struct BitBucketServerAuthor: Decodable, Equatable {
+    /// A user that is parecipating in the PR
+    public struct Participant: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
```

Comment:
<!--
  0 failure: 
  1 warning:  Types should be n...
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/BitBucketServerDSL.swift;
  Line: 301;
-->

- :warning: Types should be nested at most 1 level deep (`nesting`)